gateway-contract/readme.md

```
@@ -0,0 +1,277 @@
+### Build Individual Contracts
+
+```bash
+# Build specific contract
+cargo build --target wasm32v1-none --release -p core_contract
+cargo build --target wasm32v1-none --release -p escrow_contract
+cargo build --target wasm32v1-none --release -p factory_contract
+cargo build --target wasm32v1-none --release -p auction_contract
+```
+
+The compiled WASM files will be located in:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)